### DISCUSSION TITLE
LFX Mentorship Mentee Applications for 2024 Term 3 (Sep-Nov) Open Now!

### DISCUSSION THREAD

**Discussion by nate-double-u** [cncf/mentoring#1294] [Announcements]
Hi Everyone,

The LFX mentorship projects for 2024 Term 03 (September-November) are now listed publicly, and can be found here:
https://github.com/cncf/mentoring/blob/main/programs/lfx-mentorship/2024/03-Sep-Nov

Mentee applications are now open!

Applications will be open until Tuesday, August 13, 2024. Mentee application instructions can be found on the [Program Guidelines](https://github.com/cncf/mentoring/blob/main/programs/lfx-mentorship/README.md#program-guidelines) page.

### Timeline

| activity                                                                                                     | date                                                                                |
|--------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------|   
| project proposals                                                                                            | Jul 3 - Jul 30 5:00 PM PDT                                                          |
| mentee applications open                                                                                     | Jul 31 - Aug 13 5:00 PM PDT                                                         |
| application review/admission decisions                                                                       | Aug 14 - Aug 27 5:00 PM PDT                                                         |
| selection notifications                                                                                      | Sept 3, 2024 5:00 PM PDT                                                            |
| Mentorship program begins with the initial work assignments                                                  | Sept 9, 2024                                                                        | 
| Midterm mentee evaluations and first stipend payments                                                        | October 15, 2024 5:00 PM PDT (the week following the National Day holiday in China) |
| Final mentee evaluations and mentee feedback/blog submission due, second and final stipend payment approvals | Nov 26, 2024 5:00 PM PDT (Giving some time after KubeCon NA)                        |
| Last day of term                                                                                             | November 29, 2024                                                                   |

**Comment by nate-double-u** [comment]
We're aware that [CNCF - Inspektor Gadget: DNS/HTTP Event Generation Capabilities (2024 Term 3)](https://mentorship.lfx.linuxfoundation.org/project/c7ba30b5-625d-4086-ad09-c60071e01a8c) is not open for applicaitons yet. We're working on it and I'll let you know when you can apply.

**Reply by nate-double-u** [reply]
We're having the same issue with [CNCF - KubeEdge: Multimodal Large Model Joint Learning via KubeEdge-Ianvs Reproduction (2024 Term 3)](https://mentorship.lfx.linuxfoundation.org/project/d5d315c7-aaee-46ee-895e-a0f9e6ffed4b) & [CNCF - KubeEdge: Cloud-Edge Speculative Decoding for LLM via KubeEdge-Ianvs (2024 Term 3)](https://mentorship.lfx.linuxfoundation.org/project/bfa8251f-a975-4e07-8e7a-915df3518551). Will let you all know when it's sorted.

**Reply by nate-double-u** [reply]
These are all open for applications now, thanks for your patience! Go check them out!

**Comment by nate-double-u** [comment]
We've opened some new projects, check them out!
* [Karmada](https://github.com/cncf/mentoring/tree/main/programs/lfx-mentorship/2024/03-Sep-Nov#karmada)
* [KubeEdge](https://github.com/cncf/mentoring/tree/main/programs/lfx-mentorship/2024/03-Sep-Nov#kubeedge)
* [Meshery](https://github.com/cncf/mentoring/tree/main/programs/lfx-mentorship/2024/03-Sep-Nov#meshery)

**Comment by nate-double-u** [comment]
We've opened 4 more projects, take a look:
* [CNCF TAG Network](https://github.com/cncf/mentoring/tree/main/programs/lfx-mentorship/2024/03-Sep-Nov#cncf-tag-network)
* [Kyverno](https://github.com/cncf/mentoring/tree/main/programs/lfx-mentorship/2024/03-Sep-Nov#kyverno)

**Comment by nate-double-u** [comment]
We've opened up 3 more projects! This is the last batch, check them out!
* [OpenKruise](https://github.com/cncf/mentoring/tree/main/programs/lfx-mentorship/2024/03-Sep-Nov#openkruise)
* [Service Mesh Performance](https://github.com/cncf/mentoring/tree/main/programs/lfx-mentorship/2024/03-Sep-Nov#service-mesh-performance)
* [Kubernetes](https://github.com/cncf/mentoring/tree/main/programs/lfx-mentorship/2024/03-Sep-Nov#kubernetes)

**Comment by nate-double-u** [comment]
:loudspeaker: Remember to get your applications in for LFX 2024 Term 3! Applications are closing today at 5:00PM Pacific time
https://github.com/cncf/mentoring/tree/main/programs/lfx-mentorship/2024/03-Sep-Nov

**Comment by jayesh9747** [comment]
Hi @nate-double-u , could you tell me when the mentee for the 'Game Operation and Maintenance API' project (https://github.com/cncf/mentoring/blob/main/programs/lfx-mentorship/2024/03-Sep-Nov/README.md#game-operation-and-maintenance-api) will be selected?

**Reply by nate-double-u** [reply]
Hi, sorry for the delay. That selection has been made now and the project page updated.

**Reply by ggold7046** [reply]
Hi @nate-double-u , could you tell me what is the next date for the mentorship ?

**Reply by nate-double-u** [reply]
Hi, you can find our next term dates here: https://github.com/cncf/mentoring/tree/main/programs/lfx-mentorship/2025/01-Mar-May
